dev/devicelab/README.md

```diff
@@ -20,9 +25,12 @@ reproduce a CI test failure locally.
 To run a test, use option `-t` (`--task`):
 
 ```sh
+# from the .../flutter/dev/devicelab directory
 dart bin/run.dart -t {NAME_OF_TEST}
 ```
 
+You can see the test names in the `manifest.yaml` file in this directory or by looking in `bin/tasks/`. Do not include the `.dart` file extension. For example, `dart bin/run.dart -t complex_layout__start_up`.
```

Comment:
I'm about to add support for paths in addition to task names (which will allow you to use completion in the terminal window), so this may become outdated pretty soon.